src/main/java/core/basesyntax/file/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.file.writer;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
```

Comment:
Avoid hardcode in your solution. It's better to pass the file path as a parameter to the method, which you have done correctly. Ensure that the path provided is relative and not absolute, to maintain the portability of the code.